src/cts/src/CtsOptions.h

```diff
@@ -242,6 +242,13 @@ class CtsOptions
   {
     return sinkBufferMaxCapDerateSet_;
   }
+  void setDelayBufferDerate(float derate)
+  {
+    delayBufferDerate_ = derate;
+    delayBufferDerateSet_ = true;
+  }
+  float getDelayBufferDerate() const { return delayBufferDerate_; }
+  bool isDelayBufferDerateSet() const { return delayBufferDerateSet_; }
```

Comment:
I can't see any caller for this - is it needed?  The 1.0 default seems like it obviates the need.  (Likewise delayBufferDerateSet_ )

Reply:
If it is needed then I would use std::optional

Reply:
Removed the API